def erro_dado(mensagem):
    while True:
        try:
            return int(input(mensagem))
        except ValueError:
            print("Insira corretamente (apenas n√∫meros).")
